src/charm.py

```diff
@@ -840,17 +848,32 @@ def _run_wp_cli(self, cmd, timeout=60, combine_stderr=False):
             result = Result(0, stdout, stderr)
         except ops.pebble.ExecError as e:
             result = Result(e.exit_code, e.stdout, e.stderr)
+        return_code = result.return_code
         if combine_stderr:
-            logger.debug("Run wp-cli command: %s\noutput: %s", cmd, result.stdout)
-        else:
             logger.debug(
-                "Run wp-cli command: %s\nstdout: %s\nstderr:%s",
+                f"Run command: %s\noutput: %s, return code {return_code}",
```

Comment:
We shouldn't be using f-strings in logging output. See the contributing guide for some reasoning on why.

Same comment applies for a number of other entries here.

Reply:
Updated all f-string in logging except some variables with fixed values.